tests/suites/helpers.function

```diff
@@ -127,6 +130,23 @@ test_info;
 
 /*----------------------------------------------------------------------------*/
 /* Helper Functions */
+static int platform_setup()
+{
+#if defined(MBEDTLS_PLATFORM_C)
+    if( mbedtls_platform_setup( &platform_ctx ) != 0 )
+    {
+        return -1;
```

Comment:
Mbed TLS coding standard wise it should be ```return( -1 )```.

Reply:
Done.